src/lib/OpenEXRCore/internal_coding.h

```diff
@@ -24,8 +24,8 @@
 #    include <half.h>
 #endif
 
-#if defined(__has_include)
-#    if __has_include(<x86intrin.h>)
+#if defined(__has_include) 
+#    if __has_include(<x86intrin.h>) && defined(__x86_64__)
```

Comment:
```suggestion
#if defined(__has_include) && defined(__x86_64__)
#    if __has_include(<x86intrin.h>)
```
macOS ARM64 doesn't seem to have `intrin.h`.

Reply:
@carlocab's suggestion is correct. Also, we need stdbool.h as per 

diff --git a/src/lib/OpenEXRCore/internal_zip.c b/src/lib/OpenEXRCore/internal_zip.c
index 0192ac2a..dc32794b 100644
--- a/src/lib/OpenEXRCore/internal_zip.c
+++ b/src/lib/OpenEXRCore/internal_zip.c
@@ -11,6 +11,7 @@
 
 #include <limits.h>
 #include <stdlib.h>
+#include <stdbool.h>
 #include <string.h>
 #include <zlib.h>